__test__/mentor.test.js

```diff
@@ -11,7 +11,7 @@ describe('Get /mentor/:username/mentors', () => {
     const jwtToken = await jwt.generateToken(user);
     return request(server)
       .get('/api/mentor/john/mentors')
-      .set({ token: jwtToken })
+      .set({ authorization: jwtToken })
```

Comment:
is there a reason you changing the token to authorization????

Reply:
Yeah, the token is set as authorization header